tasks/__init__.py

```diff
@@ -53,6 +54,8 @@
 from tasks.components import lint_components, lint_fxutil_oneshot_test
 from tasks.custom_task.custom_task import custom__call__
 from tasks.fuzz import fuzz
+
+# from tasks.gitlab_helpers import pippo
```

Comment:
```suggestion
```
❓ question: ‏Feels like a leftover, is it on purpose?

Reply:
Ah thank you for the catch